dom-event-delegation/main.js

```diff
@@ -0,0 +1,19 @@
+function clickTask(event) {
+  console.log(event.target);
+  console.log(event.target.tagName);
+}
+
+var taskList = document.querySelector('.task-list');
+taskList.addEventListener('click', clickTask);
+
+taskList.addEventListener('click', function (event) {
+  var targetButton = document.querySelector('button');
+  if (event.target === targetButton || event.target.nodeName === 'BUTTON') {
```

Comment:
You only want the second expression here, there's no need to check if it is the first button in the page, you just want to see if it is a button